src/runtime_src/core/edge/skd/xrt_skd.h

```diff
@@ -83,37 +89,43 @@ class skd
   void
   fini();
 
+  void set_signal(int sig);
   void report_ready();
   void report_crash();
   void report_fini();
 
  private:
-    xclDeviceHandle parent_devHdl;
-    xclDeviceHandle devHdl;
-    xrtDeviceHandle xrtdHdl;
-    char sk_path[XRT_MAX_PATH_LENGTH];
-    uint32_t cu_idx;
-    char sk_name[PS_KERNEL_NAME_LENGTH];
-    pscontext* xrtHandle = NULL;
-    int sk_bo;
-    int sk_meta_bo;
-    unsigned char xclbin_uuid[16];
-
-    void* sk_handle;
-    void* kernel;
-    std::vector<xrt_core::xclbin::kernel_argument> args;
-    int cmd_boh = -1;
-    uint32_t *args_from_host;
-    ffi_type** ffi_args;
-    ffi_cif cif;
-    bool pass_xrtHandles = false;
-    int num_args;
-    int return_offset;
+    xclDeviceHandle m_parent_devhdl = 0;
```

Comment:
Being that there are a lot of member variables here, it might be best to group them together on how they are used and to add a comment to each grouping.